src/core.js

```diff
@@ -1579,14 +1635,28 @@ define([
 
             section.mug = mug;
             section.properties = _(section.properties)
-                .map(_getWidgetClassAndOptions)
+                .map(function (property) {
+                    return getWidgetClassAndOptions(property, mug);
+                })
                 .filter(_.identity);
 
             if (section.properties.length) {
                 this.getSectionDisplay(mug, section).appendTo($content);
             }
         }
 
+        this.$f.find(".fd-section-changer").html(section_changer({
+            sections: _.map(_.filter(_.rest(_this.getSections(mug)), function(s) {
```

Comment:
This might be a good place to use [`_.chain()`](http://underscorejs.org/#chain).